kitsune/products/migrations/0015_auto_20240722_0449.py

```diff
@@ -0,0 +1,527 @@
+        "parent_topic_slug": "settings",
+    },
+    {
+        "old_title": "Privacy and security settings",
+        "old_slug": "privacy-and-security-settings",
+        "title": "Privacy and security",
+        "slug": "privacy-and-security",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Passwords, forms and search",
+        "old_slug": "passwords-forms-and-search",
+        "title": "Passwords and sign in",
+        "slug": "passwords-and-sign-in",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Customize settings and prefences",
+        "old_slug": "customize-settings-and-prefences-5",
+        "title": "Customization",
+        "slug": "customization",
+        "parent_topic_slug": "settings",
+    },
+    {
+        "old_title": "Thunderbird controls and buttons",
+        "old_slug": "thunderbird-controls-and-buttons",
+        "title": "Settings",
+        "slug": "settings",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Calendar",
+        "old_slug": "calendar",
+        "title": "Calendar",
+        "slug": "calendar",
+        "parent_topic_slug": "email-and-messaging",
+    },
+    {
+        "old_title": "Install and update",
+        "old_slug": "install-and-update-2",
+        "title": "Installation and updates",
+        "slug": "installation-and-updates",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Thunderbird versions and languages",
+        "old_slug": "thunderbird-versions-and-languages",
+        "title": "Settings",
+        "slug": "settings",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Copy your personal information from one Thunderbird to another",
+        "old_slug": "copy-your-personal-information-from-one-thunderbird-to-another",
+        "title": "Import and export email",
+        "slug": "import-and-export-email",
+        "parent_topic_slug": "email-and-messaging",
+    },
+    {
+        "old_title": "Emails",
+        "old_slug": "emails",
+        "title": "Email and messaging",
+        "slug": "email-and-messaging",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Set up email",
+        "old_slug": "set-up-email",
+        "title": "Account management",
+        "slug": "account-management",
+        "parent_topic_slug": "accounts",
+    },
+    {
+        "old_title": "Read, send and organize emails",
+        "old_slug": "read-send-and-organize-emails",
+        "title": "Send and receive email",
+        "slug": "send-and-receive-email",
+        "parent_topic_slug": "email-and-messaging",
+    },
+    {
+        "old_title": "News Feeds (RSS), Blogs and Social",
+        "old_slug": "news-feeds-rss-blogs-and-social-thunderbird",
+        "title": "Settings",
+        "slug": "settings",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Contacts",
+        "old_slug": "contacts",
```

Comment:
I think this should be `contacts-thunderbird`. There are no old-style topics with the slug `contacts` in either stage or prod.